content/docs/1.3.0/tools/iofogctl/application-yaml-spec.md

```diff
@@ -4,6 +4,12 @@
 
 For a complete documentation of all available `iofogctl` commands, please see [our github repository](https://github.com/eclipse-iofog/iofogctl/blob/v1.3.0-beta/docs/md/iofogctl.md)
 
+### Note for those upgrading from a previous version
+
+We have updated the yml spec significantly, and while small changes in practice, your current yml files will fail to deploy on 1.3.0
```

Comment:
Missing a few word in "while small changes in practice"

Reply:
What words? Sorry, that seems fine to me, but maybe brain fart